src/Surfnet/Stepup/Identity/Event/SecondFactorVettedPossessionSkippedEvent.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+/**
+ * Copyright 2014 SURFnet bv
```

Comment:
IIRC the current convention is to use the current year in the licence. 

src/Surfnet/Stepup/Helper/SecondFactorProvePossessionHelper.php also violates this.